src/main/java/seedu/address/ui/PersonListPanel.java

```diff
@@ -49,11 +47,16 @@ public PersonListPanel(ObservableList<Person> personList, Summary summary) {
         personListView.setOnMouseClicked(new EventHandler<Event>() {
             @Override
             public void handle(Event event) {
-                Person selectedPerson = personListView.getSelectionModel().getSelectedItem();
-                logger.info("selected " + selectedPerson);
-                selectedPersonPanelPlaceholder.setVvalue(0.0);
-                selected.updatePerson(selectedPerson);
-                selected.setPersonDetails();
+                MultipleSelectionModel<Person> selectedPersonModel = personListView.getSelectionModel();
```

Comment:
Could you summarise the main reason why you used MultipleSelectionModel?

Is it mainly for abstraction purposes?

Reply:
Yes, multiple selection model is an abstraction for the display model. Perhaps i shall include more comments in the next update